TemplateBundle/content/Application/Application.App/BlankPage.fs

```diff
@@ -0,0 +1,26 @@
+namespace Application.App
+
+
+module BlankPage =
+    open Elmish
+    open Avalonia.Controls
+    open Avalonia.FuncUI.Components.Hosts
+    open Avalonia.FuncUI.Elmish
+    open Avalonia.FuncUI.DSL
+
+
+    type State =
+        { noop: bool }
```

Comment:
having a blank page sounded good but now feels a bit wrong. Maybe it could have something simple that fits into the rest of the app. I think it would be good if the whole app would have on topic or something.

Reply:
I think it looks like that because I have not added text related to the project resources and useful links

I didn't suggest a theme because I feel these are throwaway pages, once you start your new project you're very likely to either delete a page or re-do the contents
but let me see if I can get something inside and have it look better

Reply:
> I think it looks like that because I have not added text related to the project resources and useful links
forgot about that, your right.

> I didn't suggest a theme because I feel these are throwaway pages, once you start your new project you're very likely to either delete a page or re-do the contents
but let me see if I can get something inside and have it look better

Yeah, it's complicated.